src/test/java/core/basesyntax/FruitTransactionParserImplTests.java

```diff
@@ -0,0 +1,87 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTests {
+    private static FruitTransactionParser transactionParser;
+    private static List<String> testData;
+
+    @BeforeClass
+    public static void beforeClass() {
+        transactionParser = new FruitTransactionParserImpl();
+        testData = new ArrayList<>();
+    }
+
+    @Test
+    public void parseFruitTransaction_emptyData_notOk() {
+        testData.add("type,fruit,quantity");
+        List<FruitTransaction> expected = new ArrayList<>();
+        List<FruitTransaction> actual = transactionParser.parseFruitTransaction(testData);
+        assertEquals(expected, actual);
```

Comment:
```suggestion
        assertEquals(Collections.emptyList(), actual);
```
check this in other places